src/gpio.rs

```diff
@@ -457,6 +475,24 @@ where
         !self.is_set_high()
     }
 
+    /// Returns the level to which this pin is currently set
+    ///
+    /// This method is only available, if two conditions are met:
+    /// - The pin is in the GPIO state.
+    /// - The pin direction is set to output.
+    ///
+    /// See [`Pin::into_output_pin`] and [`into_output`]. Unless both of these
+    /// conditions are met, code trying to call this method will not compile.
+    ///
+    /// [`Pin::into_output_pin`]: ../pins/struct.Pin.html#method.into_output_pin
+    /// [`into_output`]: #method.into_output
+    pub fn get_set_level(&self) -> Level {
```

Comment:
Weird name, but I can't think of a better one off the top of my head. Can be left as-is for now.

Reply:
should be resolved by https://github.com/lpc-rs/lpc8xx-hal/pull/311/commits/0ec25f031de19bc2f992d829f0476bc0e8dc44bd, every pin direction uses `is_high` now